### ISSUE TITLE
[Hunter][Talent] Hunter vs. Wild

### ISSUE THREAD

**Issue by Viryll** [Stormforge-gg/Bugtracker-Fordring#1398]
**Description**: 
Talent should do this:
Increases you and your pet's attack power and ranged attack power equal to X% of your total Stamina.
![image](https://user-images.githubusercontent.com/98886485/174346701-bf194915-bdfa-488c-a05b-747727fdce0d.png)
This staff grants me 100 attack power and 133 stamina on equip. It should give per 133*0.3 + ranged attack power. It only gives 20 attack power, meaning pet doesn't get attack power from my stamina.
With sword below
Me:
![image](https://user-images.githubusercontent.com/98886485/174347136-f9ff8815-082e-451a-91af-4bf74c47d97d.png)
Pet:
![image](https://user-images.githubusercontent.com/98886485/174346898-3ae76ef6-8a49-46df-8a22-adbecf890f0e.png)
With staff below 
Me:
![image](https://user-images.githubusercontent.com/98886485/174347119-277da369-e148-4a9c-92a4-20d25551e788.png)
pet:
![image](https://user-images.githubusercontent.com/98886485/174347020-11144aac-73de-441c-ace7-6ff57c98bc52.png)

**Current behaviour**:
Pet doesn't get attack power from palyer stamina.
**Expected behaviour**: 
Pet gets attack power scaled with player stamina.

**Revision**: 
1522

**Comment by slicplaya** [comment]
This might also be tied to Why the Pet talent "Wild Hunt" isn't working at all, since the pet gains contributions from hunter stamina and atk pwr from gear. It's currently not working as well.

**Comment by Viryll** [comment]
> This might also be tied to Why the Pet talent "Wild Hunt" isn't working at all, since the pet gains contributions from hunter stamina and atk pwr from gear. It's currently not working as well.

Wild Hunt does work. My pet got 1417 from 1280 attack power. 
After dismiss, summon he goes back to 1280 tho. Until I remove and apply aspect of hawk again.

**Comment by Wolffenstein** [comment]
Fixed in rev 1538.

**Comment by Viryll** [comment]
[wowwiki]https://wowwiki-archive.fandom.com/wiki/Hunter_vs._Wild
[wotlkdb]https://wotlkdb.com/?spell=56341
**Still doesn't work, we need to re-open the issue please.**
![image](https://user-images.githubusercontent.com/98886485/174833545-b7976617-be4e-4d4c-89c5-5c3ae851a481.png)
Attack power without 132 stamina buff on owner.
![image](https://user-images.githubusercontent.com/98886485/174833608-768c8896-56a3-4f38-85ed-3bdc342a0f4c.png)
Attack power with 132 stamina buff on owner.
Difference is 11 attack power.
How it should work 132*30/100 ~39,6 + (132*1.1)*30/100*0,24 = 39,6 + 10,45
In short the talent still doesn't work.

**Revision: 1541**

**Comment by slicplaya** [comment]
Make a new ticket, he's too busy to check old closed ones.
